### PR TITLE
Qradar v2- error is not presented on war room entry when command fails

### PR DESCRIPTION
**Pull request by tallieber**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/35103

## Description
Remove an import of demisto_mock as demisto

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/QRadar/ReleaseNotes/1_2_13.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### IBM QRadar v2
+- Fix an issue with error occurrence, war room entry did not present the error description.
```
Comment (bakatzir) [inline Packs/QRadar/ReleaseNotes/1_2_13.md:4]:
```suggestion
- Fix an issue with error occurrence, war room entry did not present the error description.
- Updated the Docker image to: *demisto/python3:3.9.2.17957*.
```

**Review by bakatzir — COMMENTED** [review]
@tallieber if you do not add a UT please ref a print screen of the sol(here or in the issue if it holds customer details)

**Comment by ShahafBenYakir** [comment]
The fix for the issue is removing the `import demisto_mock as demisto`? Can you explain how this creates an issue (under the assumption that this line is being removed before the integration is delivered to production) 
Settings 2021-03-29 16-37-02

**Comment by tallieber** [comment]
> The fix for the issue is removing the `import demisto_mock as demisto`? Can you explain how this creates an issue (under the assumption that this line is being removed before the integration is delivered to production)
> Settings 2021-03-29 16-37-02

I had a mistake... I used an integration that was configured in the local host and now I talked with @DeanArbel I understand there was a bug fix with exactly the same issue. I remove this PR and close the bug because they just need to update the pack.

**Pull request closed without merging**